test/runtests.jl

```diff
@@ -112,10 +112,10 @@ using Test
         @test include("MLJ/integration.jl")
     end
 
-    @info "API"
-    @test include("api.jl")
+    # @info "API"
```

Comment:
Was this just a left over that you left commented out?

Reply:
Oops. Just re-instated those. Hopefully that will not effect CI.

Reply:
Looks like we're good to go.